### DISCUSSION TITLE
Connect to Oracle

### DISCUSSION THREAD

**Discussion by Flosckow** [collerek/ormar#302] [General]
Hello, I'm very interested in whether there will be oracle support?

**Comment by collerek** [comment]
There will be when I switch to sqlalchemy from databases. Databases were basically abandoned, there is no active maintainer with merge permission as of now and changes in the code base are super slow.

But that is quite a big task so it will be in the future (hard to tell when exactly)

**Reply by Flosckow** [reply]
Okay, thanks for you  reply.
